docs/README.md

```diff
@@ -0,0 +1,26 @@
+Table of Contents
+
+- [Introduction](index.md)
```

Comment:
I wanted to make sure we remember the [comment discussion about the outline starting with `overview` and `installation-guide`](https://github.com/asyncapi/generator/pull/793#issuecomment-1154620376). 

I'm not sure about **introduction** and going straight into **asyncapi spec file**.  What do you think about the following instead?
- `docs/tools/generator/overview`
- `docs/tools/generator/installation-guide`

Reply:
Overview seems good. Maybe we can have overview and installation-guide in the same file🤔 Since installation guide is just few lines?

Reply:
Hmm I see what you mean! 🤔 So, I thought about it and it seems off, because then in the **left-hand** navigation of docs the user would only see "overview" but not the "installation guide" because headers don't show up in that area.

How do you feel about doing it this way, instead? (It would also mirror consistency with what we're doing in our **Tutorials** now, as you can see in our [updated IA outline for Streetlights tutorial](https://www.asyncapi.com/docs/tutorials/streetlights).)
- `tools/generator/set-up-generator-tool`
- `tools/generator/set-up-generator-tool#introduction`
- `tools/generator/set-up-generator-tool#background-context`
- `tools/generator/set-up-generator-tool#installation-guide`

Reply:
I also wanted you to take a look at a similar scenario and how **Gatsby** handled it, because I think it will visually also help us for our work in this project. 😀
<img width="1366" alt="Screen Shot 2022-06-29 at 1 27 53 PM" src="https://user-images.githubusercontent.com/19964402/176538680-1fa3058c-4a12-472f-98b8-11695cbce511.png">

Reply:
Other than this particular point we're pondering, the rest of the outline you have proposed here seems to work pretty great! 

Of course, make sure that Lukasz signs off on the order of the pages, since he will know best technically the correct order needed for users.

Reply:
The main reason I was refraining from using the word "Overview" is because we already have an Overview section in the Tools, here it is:

![Screenshot (86)](https://user-images.githubusercontent.com/77961530/176629640-fc51c0a7-4fdb-4f9d-bb3b-055238085098.png)

If we include the generator overview again, we will have two overview's in the same section and most likely one below the other🤔
your suggested outlines and the gatsby reference sound awesome 
- tools/generator/set-up-generator-tool
- tools/generator/set-up-generator-tool#introduction
- tools/generator/set-up-generator-tool#background-context
- tools/generator/set-up-generator-tool#installation-guide
- tools/generator/set-up-generator-tool#summary

But I have to take @derberg 's opinion on whether these outlines sound technically correct or not😅

Reply:
remember folks, try not to think to much about the website, content first 😄 

I'm ok with `introduction` or `overview`. There is no standard out there, people do it differently, and I, as a user, stopped paying attention to the `title`. Just leave `Introduction` and we can always improve at the end, current README navigation is there only temporarily until we integrate with the website

Reply:
Got it! Thanks @derberg 
Thanks @alequetzalli for the Gatsby reference! I checked out their documentation and that section about "setting up" would help the generator documentation greatly in future PR's

Reply:
> Thanks @alequetzalli for the Gatsby reference! I checked out their documentation and that section about "setting up" would help the generator documentation greatly in future PR's

@pratik2315 @Florence-Njeri Great! So it sounds like we agree with this new outline moving forward!! 

I **completely agree with you** that using the term _overview_ in this section would be over use. That's why I am glad that we both agree that the word _introduction_ is better suited for this section.